modules/ROOT/pages/_partials/mmp-deploy-to-rtf.adoc

```diff
@@ -0,0 +1,281 @@
+//= Deploy Applications to Runtime Fabric Using the Mule Maven Plugin
```

Comment:
Is the **partial** modules/ROOT/pages/_partials/mmp-deploy-to-rtf.adoc needed in **v4.4**?

Reply:
@Cristian-Venticinque

Reply:
Yes, because I re-structured this article to be reused both in mule and RTF docs.